pkg/task/get.go

```diff
@@ -1,31 +1,65 @@
 package task
 
 import (
-	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
 
-	"github.com/puppetlabs/nebula-sdk/pkg/taskutil"
 	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/evaluate"
 )
 
 func (ti *TaskInterface) ReadData(path string) ([]byte, error) {
-	eval, err := taskutil.EvaluatorFromDefaultPlan(ti.opts)
+	u, err := url.Parse(ti.opts.SpecURL)
 	if err != nil {
 		return nil, err
 	}
 
+	q, err := url.ParseQuery(u.RawQuery)
+	if err != nil {
+		return nil, err
+	}
+
+	if path != "" {
+		q.Add("q", path)
+		q.Add("lang", "jsonpath-template")
```

Comment:
This requires an update to the tasks spec handler to control the query language.

Reply:
Done in https://github.com/puppetlabs/nebula-tasks/pull/113